### PR TITLE
Overall layout updates.

### PR DESCRIPTION
**Pull request by biocom**
Fixing #630 

_(default state -> wrapping when large digits are present)_
Screenshot 2020-02-27 at 11 47 57

_(pending)_
Screenshot 2020-02-27 at 11 47 37

_(hover state)_
Screenshot 2020-02-27 at 11 47 35

_(default state -> small digit)_
Screenshot 2020-02-27 at 11 47 31

Fixing #628 
Screenshot 2020-02-27 at 12 54 51

Screenshot 2020-02-27 at 12 54 01
Screenshot 2020-02-27 at 12 53 32
Screenshot 2020-02-27 at 12 53 20

Fixing #623 
Screenshot 2020-02-27 at 10 19 29

Fixing #589 (Created a `ContentPage` wrapper inheriting from PageWrapper for this, as a exported const)
Screenshot 2020-02-27 at 13 25 58

Screenshot 2020-02-27 at 13 26 05

**Also included**
- Applied a default font-size to the <main> element so we have a sensible fallback font size.
- Disabled autocomplete on the Tradewidget form.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr635--dexreact.review.gnosisdev.com

**Comment by biocom** [comment]
Also optimise the error message page (center content):
Screenshot 2020-02-27 at 13 55 08

File: src/components/TradeWidget/Price.tsx [hunk 1]
```diff
@@ -89,6 +102,10 @@ const PriceInputBox = styled.div`
       color: #218dff;
     }
 
+    &:focus::placeholder {
+      color: transparent;
```
Comment (Velenir) [inline src/components/TradeWidget/Price.tsx]: This is cool but htere are no placeholders in PriceInputBox > input
Planning to add some?

Comment (biocom) [reply src/components/TradeWidget/Price.tsx]: This was an attempt to implement a CSS only solution where the placeholder (usually a '0') is visually removed once you focus on it.

But indeed, there didn't seem to exist a placeholder when I fully tested it, but just left the styles in place.

Yes, if we can implement the default 0's as placeholders that would be cool.

File: src/components/PoolingWidget/AddFunding.styled.ts [hunk 2]
```diff
@@ -24,14 +24,14 @@ export const AddFundingWrapper = styled.div`
     > span {
       padding: 0 1rem;
       box-sizing: border-box;
-      font-size: 1.5rem;
-      line-height: 1.4;
+      font-size: inherit;
+      line-height: inherit;
```
[the remark this answers is not in the document]
Comment (biocom) [reply src/components/PoolingWidget/AddFunding.styled.ts]: Yes, some are probably redundant and I might test/cleanup those. Thanks :)

**Comment by biocom** [comment]
Also included in this update is the ORDERS page on mobile, now uses a proper white background. But only on mobile. When nested inside the OrdersPanel (tablet/desktop) these styles aren't applied.

File: src/components/TradeWidget/Price.tsx (hunk 1 above)
Comment (Velenir) [reply src/components/TradeWidget/Price.tsx]: This is late, but btw `visibility: hidden` would also work

**Pull request merged**